src/string.c

```diff
@@ -12,4 +12,39 @@ size_t strlen(char *string)
   }
 
   return len;
-}
\ No newline at end of file
+}
+
+i32 strcmp(char *str1, char *str2)
```

Comment:
If you didn't write this code please add some kind of a credit somewhere.

Reply:
I wrote I based off documentation